snaver/views.py

```diff
@@ -51,3 +51,50 @@ def get_queryset(self):
         )
 
         return subcategory_details
+
+
+class ChartsListView(ListView):
```

Comment:
A lot of code is repeated here. Not sure how to properly fix it.

Reply:
I am aware of it. Let's accept it now and improve it later. I will be working on the charts in the next sprint, so we will improve it later.